main.go

```diff
@@ -69,6 +70,11 @@ func init() {
 		model.OktetoBinImageTag = bin
 		log.Infof("using %s as the bin image", bin)
 	}
+	if model.FileExists(".env") {
```

Comment:
I think I would do this when initializing the okteto context

Reply:
yep, I think it would makes sense to do it as part of the context initialization